src/output.h

```diff
@@ -17,6 +17,11 @@
 #define MAX_FILE_BYTES 65536
 #define MAX_LOT_FRAGMENTS (MAX_FILE_BYTES / LOT_FRAGMENT_SIZE)
 
+#define AUDIO_FRAME_CHANNELS 2
+#define AUDIO_FRAME_LENGTH (NRSC5_AUDIO_FRAME_SAMPLES * AUDIO_FRAME_CHANNELS)
+
+#define OUTPUT_BUFFER_LENGTH (64 * AUDIO_FRAME_LENGTH)
```

Comment:
Where does 64 come from? Is it `MAX_AUDIO_PACKETS` or something else?

Reply:
64 is technically an arbitrary number. I chose the output buffer to be 64 frames. This is due to the flaw that the output buffer is limited in size. I can switch to a linked list approach. That would require more code when splitting frames to output at a fixed ratio. Wanted the code to be simple.

Reply:
OK, if it's unrelated to `MAX_AUDIO_PACKETS` then it's best to stay as it is. You could perhaps give this 64 its own name.

I'll have a look over the rest of the design to see what I think of linked list vs. buffer.